Core/composer.json

```diff
@@ -17,11 +17,12 @@
         "phpunit/phpunit": "^9.0",
         "phpspec/prophecy-phpunit": "^2.0",
         "squizlabs/php_codesniffer": "2.*",
-        "phpdocumentor/reflection": "^5.0",
+        "phpdocumentor/reflection": "^5.3.3",
+        "phpdocumentor/reflection-docblock": "^5.3",
         "erusev/parsedown": "^1.6",
         "google/gax": "^1.26.0",
         "opis/closure": "^3",
-        "google/cloud-common-protos": "^0.4"
+        "google/cloud-common-protos": "~0.5"
```

Comment:
What is the reason we are using a `~` instead of `^`?

`~0.5` would mean `>=0.5 and <1.0` and `^0.4` would also mean `>=0.5 and <1.0`.

Reply:
With `^` used in a pre-1.0 release, the next minor release is NOT upgraded. So `^0.4` would mean `>=0.4.0 and <0.5.0`

Reply:
We could keep it at `^0.5`, we'll just have to upgrade it when we create new versions of `google/cloud-common-protos`... which is fine, but a little tedious.

Reply:
Understood, thanks.